main.go

```diff
@@ -1,110 +1,111 @@
+				if err != nil {
+					log.Println("Error: could not parse post body")
+					return
+				}
+				s.SendMessage("/events/room-"+roomNumber, sse.SimpleMessage(string(messageData)))
+
+			}
+		} else {
+			http.NotFound(w, r)
+		}
 
-func pushMessage(b *Broker, t template.Template, rid string) {
-	var tpl bytes.Buffer
-	t.Execute(&tpl, strings.Join([]string{"room"}, rid))
-	b.messages <- tpl.String()
+	})
+
+	// prefixToRemove := fmt.Sprintf("/rooms/%s/", numberToRemove[1])
+	// fmt.Println("removing prefix: ", prefixToRemove)
+
+	// http.StripPrefix(prefixToRemove, http.FileServer(http.Dir("html_template")))
+	// http.StripPrefix("/rooms/", http.FileServer(http.Dir("html_template")))
 }
 
-// Main routine
 func main() {
-	// Make a new Broker instance
-	b := &Broker{
-		make(map[chan string]bool),
-		make(chan (chan string)),
-		make(chan (chan string)),
-		make(chan string),
-	}
-
-	// Start processing events
-	b.Start()
-
-	// Make b the HTTP handler for "/events/".  It can do
-	// this because it has a ServeHTTP method.  That method
-	// is called in a separate goroutine for each
-	// request to "/events/".
-	http.Handle("/events/", b)
-
-	// Routing handler
-	http.HandleFunc("/rooms/", func(w http.ResponseWriter, r *http.Request) {
-		switch r.Method {
-		case http.MethodGet:
-			contentGetter(w, r, b)
-			return
-		case http.MethodPost:
-			changeContent(w, r, b)
-			return
-		default:
-			w.WriteHeader(http.StatusMethodNotAllowed)
-			return
-		}
+	s := sse.NewServer(&sse.Options{
+		Headers: map[string]string{
+			"Access-Control-Allow-Origin":  "*",
+			"Access-Control-Allow-Methods": "GET, OPTIONS",
+			"Access-Control-Allow-Headers": "Keep-Alive,X-Requested-With,Cache-Control,Content-Type,Last-Event-ID",
+		},
+		// Custom channel name generator
+		ChannelNameFunc: func(request *http.Request) string {
+			return request.URL.Path
+		},
+		Logger: log.New(os.Stdout, "go-sse: ", log.Ldate|log.Ltime|log.Lshortfile),
 	})
+	defer s.Shutdown()
+
+	// Create internal redirect server to render static files
+	http.Handle("/rooms/", handleRooms(http.FileServer(http.Dir("html_template")), s))
+
+	// Register /events endpoint
+	http.Handle("/events/", s)
+
+	// go func() {
+	// 	for {
+	// 		stringJSON := `{"title": "Title for Room 1", "next": "Next Presentation title @ 16:00", "time": "` + time.Now().String() + `"}`
+	// 		s.SendMessage("/events/channel-1", sse.SimpleMessage(stringJSON))
+	// 		time.Sleep(5 * time.Second)
+	// 	}
+	// }()
+
+	// go func() {
+	// 	i := 0
+	// 	for {
+	// 		i++
+	// 		stringJSON := `{"title": "Title for Room 2", "next": "Next Presentation With a Relative Big Text Title, so I Can Test This big Text @ 16:00", "time": "` + strconv.Itoa(i) + `"}`
+	// 		s.SendMessage("/events/channel-2", sse.SimpleMessage(stringJSON))
+	// 		time.Sleep(5 * time.Second)
+	// 	}
+	// }()
 
-	// Routing handler
-	http.Handle("/", http.FileServer(http.Dir("static_files")))
+	fmt.Println("============= Instructions ==============")
```

Comment:
And what about moving this to the README file? Or do you prefer having an actual CLI displaying this information?